stable/appmesh-controller/README.md

```diff
@@ -127,13 +127,27 @@ https://github.com/aws/aws-app-mesh-examples/blob/5a2d04227593d292d52e5e2ca638d8
 ``` 
 
 #### Without IRSA   
-If not setting up IAM role for service account, apply the IAM policies manually to your worker nodes:
+If not setting up IAM role for service account, apply the IAM policies manually to your eks worker nodes. 
+**Note:** You can use the service account for controller iam policy from above steps, but directly apply the envoy iam policy to the worker nodes.
```

Comment:
Could you change this wording to somthing like this : "You can use the service account for controller iam policy from above steps or directly apply the controller iam policy to worker nodes as you would do for envoy iam policy"

Reply:
ACK